### PR TITLE
feat(dedicated): add vrack orderable in  ipv4

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-14886 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

Allow Vrack while ordering an additional ipv4.

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js [hunk 1]
```diff
@@ -107,6 +107,7 @@ export default class AgoraIpV4OrderController {
       .all({
         user: this.User.getUser(),
         services: this.Ipv4AgoraOrder.getServices(),
+        vrackServices: this.Ipv4AgoraOrder.getVrackService(), // toDo
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js:110]:
```suggestion
        vracks: this.Ipv4AgoraOrder.getVrackService(), // toDo
```

Don't use vrackServices term as it correspond to another product

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js [hunk 2]
```diff
@@ -115,7 +116,7 @@ export default class AgoraIpV4OrderController {
           type: 'PARKING',
         });
         this.user = results.user;
-        this.services = results.services;
+        this.services = [...results.services, ...results.vrackServices];
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js:119]:
```suggestion
        this.services = [...results.services, ...results.vracks];
```

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.service.js [hunk 3]
```diff
@@ -21,6 +22,24 @@ export default class Ipv4AgoraOrder {
     });
   }
 
+  getVrackService() {
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.service.js:25]:
```suggestion
  getVracks() {
```

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.service.js [hunk 4]
```diff
@@ -21,6 +22,24 @@ export default class Ipv4AgoraOrder {
...
+    return this.iceberg('/vrack')
+      .query()
+      .expand('CachedObjectList-Pages')
+      .execute()
+      .$promise.then(({ data }) =>
+        data.map((vrackService) => {
+          const serviceName = vrackService.iam.urn.split(':').pop();
+          const displayName = vrackService.name || serviceName;
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.service.js:33]:
```suggestion
        data.map((vrack) => {
          const serviceName = vrack.iam.urn.split(':').pop();
          const displayName = vrack.name || serviceName;
```

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js (hunk 1 above)
Comment (sachinrameshn) [reply packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js]: done

**Review by qpavy — DISMISSED** [review]

**Review by mhelhali-soufien — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — DISMISSED** [review]

**Review by SimonChaumet — DISMISSED** [review]

**Review by SimonChaumet — APPROVED** [review]

**Review by mhelhali-soufien — APPROVED** [review]

**Pull request merged**